src/onINP.ts

```diff
@@ -57,20 +63,88 @@ const getInteractionCountForNavigation = () => {
 const MAX_INTERACTIONS_TO_CONSIDER = 10;
 
 // A list of longest interactions on the page (by latency) sorted so the
-// longest one is first. The list is as most MAX_INTERACTIONS_TO_CONSIDER long.
+// longest one is first. The list is at most MAX_INTERACTIONS_TO_CONSIDER long.
 let longestInteractionList: Interaction[] = [];
 
 // A mapping of longest interactions by their interaction ID.
 // This is used for faster lookup.
-const longestInteractionMap: {[interactionId: string]: Interaction} = {};
+const longestInteractionMap: {[interactionId: string]: Interaction} =
+  Object.create(null);
```

Comment:
Genuine curiosity: is there a benefit to this change from `{}` (which has less characters)?